modules/ossm-installing-external-control-plane.adoc

```diff
@@ -0,0 +1,465 @@
+----
+$ oc get project external-istiod --context "${CTX_DATA_PLANE_CLUSTER}" || oc new-project external-istiod --context "${CTX_DATA_PLANE_CLUSTER}"
+----
+
+.. Create an `Istio` resource on the data plane cluster by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc --context "${CTX_DATA_PLANE_CLUSTER}" apply -f -
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: external-istiod
+spec:
+  version: v${ISTIO_VERSION}
+  namespace: external-istiod
+  profile: remote
+  values:
+    defaultRevision: external-istiod
+    global:
+      remotePilotAddress: ${EXTERNAL_ISTIOD_ADDR}
+      configCluster: true <1>
+    pilot:
+      configMap: true
+      istiodRemote:
+        injectionPath: /inject/cluster/cluster2/net/network1
+EOF
+----
+<1> This setting identifies the data plane cluster as the source of the mesh configuration.
+
+. Create a project called `istio-cni` on the data plane cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get project istio-cni --context "${CTX_DATA_PLANE_CLUSTER}" || oc new-project istio-cni --context "${CTX_DATA_PLANE_CLUSTER}"
+----
+
+.. Create an `IstioCNI` resource on the data plane cluster by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc --context "${CTX_DATA_PLANE_CLUSTER}" apply -f -
+apiVersion: sailoperator.io/v1
+kind: IstioCNI
+metadata:
+  name: default
+spec:
+  version: v${ISTIO_VERSION}
+  namespace: istio-cni
+EOF
+----
+
+. Set up the external {istio} control plane on the control plane cluster:
+
+.. Create a project called `external-istiod` on the control plane cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get project external-istiod --context "${CTX_CONTROL_PLANE_CLUSTER}" || oc new-project external-istiod --context "${CTX_CONTROL_PLANE_CLUSTER}"
+----
+
+.. Create a `ServiceAccount` resource on the control plane cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context="${CTX_CONTROL_PLANE_CLUSTER}" create serviceaccount istiod-service-account -n external-istiod
+----
+
+.. Store the API server address for the data plane cluster in an environment variable by running the following command:
++
+[source,terminal]
+----
+$ DATA_PLANE_API_SERVER=https://<hostname_or_IP_address_of_the_API_server_for_the_data_plane_cluster>:6443
+----
+
+.. Install a remote secret on the control plane cluster that provides access to the API server on the data plane cluster by running the following command:
++
+[source,terminal]
+----
+$ istioctl create-remote-secret \
+  --context="${CTX_DATA_PLANE_CLUSTER}" \
+  --type=config \
+  --namespace=external-istiod \
+  --service-account=istiod-external-istiod \
+  --create-service-account=false \
+  --server="${DATA_PLANE_API_SERVER}" | \
+  oc --context="${CTX_CONTROL_PLANE_CLUSTER}" apply -f -
+----
+
+.. Create an `Istio` resource on the control plane cluster by running the following command:
++
+[source,terminal]
```

Comment:
If you wanted to be fancy you could turn this into a collapsible block to reduce scrolling for the user. [%collapsible]